src/test/java/uk/gov/ons/census/fwmt/tests/acceptance/steps/household/HouseholdSteps.java

```diff
@@ -42,11 +42,13 @@
 public class HouseholdSteps {
 
   private static final String RM_CREATE_REQUEST_RECEIVED = "RM_CREATE_REQUEST_RECEIVED";
-  private static final String COMET_CREATE_SENT = "COMET_CREATE_SENT";
-  private static final String CANONICAL_CANCEL_RECEIVED = "CANONICAL_CANCEL_RECEIVED";
+  private static final String RM_UPDATE_REQUEST_RECEIVED = "RM_UPDATE_REQUEST_RECEIVED";
+  private static final String RM_CANCEL_REQUEST_RECEIVED = "RM_CANCEL_REQUEST_RECEIVED";
+  private static final String COMET_CREATE_JOB_REQUEST = "Comet - Create Job Request";
```

Comment:
Make these the same as the Field name, like all the others

Reply:
This has been done